stress/common/src/main/java/alluxio/stress/fuse/FuseIOOperation.java

```diff
@@ -17,6 +17,8 @@
 public enum FuseIOOperation {
   /** Write operation to test the write throughput or prepare data for reading. */
   WRITE("Write"),
+  /** List the files to cache metadata on client side, separating metadata and data performance. */
```

Comment:
If client-side metadata cache is enabled, listing files before reading can help cache the metadata and gain more accurate reading performance.